packages/design-system-docs/src/styles/components/_UsaBanner.scss

```diff
@@ -35,7 +35,9 @@
 }
 
 .c-usa-banner__button {
-  @extend %link-darker;
+  /* stylelint-disable scss/at-extend-no-missing-placeholder */
+  @extend .ds-c-link;
```

Comment:
This is the one visual change, the USA button link is now regular link blue, matching USWDS

Reply:
Nice fix here!